### ISSUE TITLE
[doc] documentação: criar um padrão para os arquivos em md

### ISSUE THREAD

**Issue by intrepidocabral** [guilhermednt/login-cidadao#23]
exemplo: 

![selecao_113](https://cloud.githubusercontent.com/assets/641411/22896145/20ae3f70-f206-11e6-9ba5-c1fdf5c9f1e8.png)

**Comment by guilhermednt** [comment]
Não seria redundante prefixarmos todos arquivos com `lc_`? Se todos tem o mesmo prefixo o resultado final será o mesmo.

Quanto ao "segundo nível" de classificação, estou com dificuldade para identificar como agrupar os arquivos que temos. Alguma sugestão?

**Comment by intrepidocabral** [comment]
> Não seria redundante prefixarmos todos arquivos com lc_? Se todos tem o mesmo prefixo o resultado final será o mesmo.

Sim. No caso de ter apenas arquivos do próprio projeto, acho que é redundante mesmo. Já vi projetos que usam uma diferenciação para doc_user e doc_dev... algo nesse sentido. 


> Quanto ao "segundo nível" de classificação, estou com dificuldade para identificar como agrupar os arquivos que temos. Alguma sugestão?

Acho que tem agrupamentos em grandes níveis que ajudam a não dispersar as coisas e a dividir os conteúdos. Por exemplo: documentação de deploy tem por objetivo prover anunciados que digam diretamente respeito a instalação. Documentação de configurações dizem respeito a enunciados complementares, como por exemplo configs do nginx, configs de url da aplicação, etc. Seria mais ou menos isso. Dividir conteúdo pra não criar aqueles tripas gigantes e ao mesmo tempo agregar por temas. Temas possíveis: 

* configs
* deploy
* develop
* template
* doc
* user
* api
* plugins

**Comment by vitor-kato** [comment]
OK para essa issue, apenas continuar mantendo o padrão nos arquivos. Vou deixar aberta  para futuras referências.
